circle.yml

```diff
@@ -36,7 +36,7 @@ stages:
         command: |
           apt-get update && apt-get -yq install build-essential
           if [ ! -x "$(which yarn)" ] ; then
-            npm -i -g yarn
```

Comment:
Oopies. Good catch!

By the way, I think what is happening here is that some of the CircleCI containers are using old images while some are using new that have yarn. It's real goofy.